.travis.yml

```diff
@@ -3,12 +3,13 @@ env:
     - CC_TEST_REPORTER_ID=97ada6a78eeebe77dc11dacfb31238d8c5c3ba39ccae8dec76279e8176221da5
 language: ruby
 rvm:
-  - 2.4
+  - 2.6.4
 before_script:
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
 script:
   - bundle exec rspec
+  - bundle exec rubocop
```

Comment:
> Seems like this is blocked by `simplecov` updating their version of rubocop. Maybe we add rubocop, but don't require in Travis? 🤔

Sounds like a sensible approach to me! We can add it back in when simplecov bumps rubocop. I'm not sure if there's a way to resolve this issue atm but will do some digging as well.

Reply:
Cool, I’ll update this accordingly.